gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeSummaryTablePanel.java

```diff
@@ -131,11 +130,11 @@ public void onClick(final AjaxRequestTarget target) {
 		addOrReplace(new WebMarkupContainer("weightColumnHeader")
 				.setVisible(categoriesEnabled && isCategoryWeightEnabled && this.isGroupedByCategory));
 
-		boolean catColVisible = categoriesEnabled && !isGroupedByCategory;
+		final boolean catColVisible = categoriesEnabled && !this.isGroupedByCategory;
 		addOrReplace(new WebMarkupContainer("categoryColumnHeader").setVisible(catColVisible));
 
 		addOrReplace(new WebMarkupContainer("dateColumnHeader")
-				.add(AttributeAppender.append("class", catColVisible ? "col-md-1" : "col-md-2"))); // steal width from date column to give to category column
+				.add(AttributeModifier.append("class", catColVisible ? "col-md-1" : "col-md-2"))); // steal width from date column to give to category column
```

Comment:
No I didn't, sorry I didn't even notice this. Will fix it now. It must have been my preferences file although I don't know why it would be changing that... going to turn off my pref file for sakai from now on, it is too hard to keep track of all the cleanup. Hopefully there will be a sakai preferences file in future.